compose.yml

```diff
@@ -244,3 +244,7 @@ services:
       ASPNETCORE_ENVIRONMENT: Production
       ConnectionStrings__EvDatabase: 'Server=db;Database=cdp;Username=cdp_user;Password=cdp123;'
       Sqs__ServiceURL: "http://localstack:4566"
+      InboundQueue__Name: "ev-inbound"
+      InboundQueue__MaxNumberOfMessages: 20
+      InboundQueue__WaitTimeSeconds: 20
```

Comment:
We only override things that are different on Docker environment to limit amount of config to maintain. 

`InboundQueue__MaxNumberOfMessages` and `InboundQueue__WaitTimeSeconds` aren't necessary here. 

Acually, I mislead you before . Even `InboundQueue__Name` isn't needed here. Sorry! Please remove all three.

Reply:
All four, there's the outbond one below.

Reply:
Ok they've been removed.